packages/lesswrong/components/sequences/TopPostsPage.tsx

```diff
@@ -830,7 +830,7 @@ function TopSpotlightsSection({classes, yearGroupsInfo, sectionsInfo, reviewWinn
 
   const filteredSpotlights = filterWhereFieldsNotNull(filteredReviewWinnersForSpotlights, 'spotlight').map(post => ({
     ...post.spotlight,
-    document: post,
+    document: { ...post, __typename: 'Post' as const },
```

Comment:
?

Reply:
Needed because filteredSpotlights get run through `getSpotlightUrl`, which now relies on `__typename` rather than `documentType` to discriminate between what type of document it is (which it needs to do to now that we've added the `tag` case)